src/ray/core_worker/transport/direct_task_transport.cc

```diff
@@ -476,7 +476,7 @@ void CoreWorkerDirectTaskSubmitter::RequestNewWorkerIfNeeded(
             // A lease request to a remote raylet failed. Retry locally if the lease is
             // still needed.
             // TODO(swang): Fail after some number of retries?
-            RAY_LOG(INFO)
+            RAY_LOG_EVERY_MS(INFO, 5000)
                 << "Retrying attempt to schedule task at remote node. Try again "
```

Comment:
is it possible to add a bit more metadata here btw (e..g, like which task is this)? Looks like the error message is a bit useless now